sqldelight-gradle-plugin/src/test/fixtures/well-formed-selects/expected/com/sample/HockeyPlayerModel.java

```diff
@@ -185,6 +185,50 @@
   @NonNull
   HockeyPlayer.Position position();
 
+  interface Select_allModel {
+    HockeyPlayerModel hockey_player();
+
+    TeamModel team();
+  }
+
+  interface For_teamModel {
+    HockeyPlayerModel hockey_player();
+
+    TeamModel team();
+  }
+
+  interface Subquery_joinModel {
+    long _id();
+
+    int age();
+  }
+
+  interface Select_expressionModel {
+    String first_name();
+
+    long count();
+  }
+
+  interface Expression_subqueryModel {
+    HockeyPlayerModel hockey_player();
+
+    long size();
+  }
+
+  interface Some_joinModel {
+    HockeyPlayerModel hockey_player();
+
+    TeamModel team();
+  }
+
+  interface With_queryModel {
```

Comment:
yes, because the calling site requires the string. Ex:

```java
// TestModel.withQuery is static string
// Test.Factory.bindWithQuery() returns the String[] containing the args.
db.rawQuery(TestModel.withQuery, Test.Factory.bindWithQuery(arg1, arg2))
```

even for `SQLiteStatement` you have to use the string to pre-compile the statement

```java
SQLiteStatement insert = db.compileStatement(TestModel.insert);

// bindInsert mutates the given SQLiteStatement and returns it.
Test.Factory.bindInsert(insert, arg1, arg2, arg3).insert();
```